selfservice/strategy/oidc/types.go

```diff
@@ -40,8 +46,10 @@ func NewCredentials(provider, subject string) (*identity.Credentials, error) {
 }
 
 type ProviderCredentialsConfig struct {
-	Subject  string `json:"subject"`
-	Provider string `json:"provider"`
+	Subject               string `json:"subject"`
+	Provider              string `json:"provider"`
+	EncryptedAccessToken  []byte `json:"encrypted_access_token"`
+	EncryptedRefreshToken []byte `json:"encrypted_refresh_token"`
```

Comment:
Make these a string as `[]byte` is base64 encoded by `encoding/json`

Reply:
I did but I don't know why maybe a mistake on my side but wrap and unwrap the encrypted data goes wrong.

Reply:
Yes just retried this and it did work due to non-printable character